docs/spring-boot/core-starter.md

```diff
@@ -0,0 +1,257 @@
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>bdk-core-spring-boot</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>bdk-core-spring-boot</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.symphony.platformsolutions</groupId>
+            <artifactId>symphony-bdk-core-spring-boot-starter</artifactId>
+            <version>1.3.0.BETA</version>
+        </dependency>
+    </dependencies>
+</project>
+```
+The following listing shows the `build.gradle` file that is created when you choose Gradle:
+```groovy
+plugins {
+    id 'org.springframework.boot' version "2.3.3.RELEASE"
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter:1.3.0.BETA'
+}
+```
+
+## Create a Simple Bot Application
+As a first step, you have to initialize your bot environment through the Spring Boot `src/main/resources/application.yaml` file: 
+```yaml
+bdk:
+    host: acme.symphony.com
+    bot:
+      username: bot-username
+      privateKeyPath: /path/to/rsa/privatekey.pem
+``` 
+> You can notice here that the `bdk` property inherits from the [`BdkConfig`](https://javadoc.io/doc/com.symphony.platformsolutions/symphony-bdk-core/latest/com/symphony/bdk/core/config/model/BdkConfig.html) class.
+
+As required by Spring Boot, you have to create an `src/main/java/com/example/bot/BotApplication.java` class:
+```java
+@SpringBootApplication
+public class BotApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+}
+```
+
+Now you can create a component for a simple bot application, as the following listing (from `src/main/java/com/example/bot/HelloBot.java`) 
+shows:
+```java
+@Component
+public class HelloBot {
+
+  @Autowired
+  private MessageService messageService;
+
+  @EventListener
+  public void onMessageSent(RealTimeEvent<V4MessageSent> event) {
+    this.messageService.send(event.getSource().getMessage().getStream(),"<messageML>Hello!</messageML>");
+  }
+}
+``` 
+
+You can finally run your Spring Boot application and verify that your bot always replies with `Hello!`. 
+
+## Subscribe to Real Time Events
+The Core Starter uses [Spring Events](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/ApplicationEventPublisher.html) 
+to deliver Real Time Events. 
+
+You can subscribe to any Real Time Event from anywhere in your application by creating a handler method that has to 
+respect two conditions: 
+- be annotated with [@EventListener](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/event/EventListener.html) 
+- have `com.symphony.bdk.spring.events.RealTimeEvent<T>` parameter
+
+Here's the list of Real Time Events you can subscribe:
+```java
+@Component
```

Comment:
as a BDK user is there a way to discover all the events I can use (they don't all inherit the same class)?